src/App.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import AddItem from './components/AddItem/AddItem.js';
+import List from './components/List/List.js';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import NavBar from './components/NavBar/NavBar.js';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <NavBar />
```

Comment:
Nit-picky comment (i.e. not super important): 
As the `NavBar` is at the bottom, maybe we should move the component to the bottom just so that the code matches up to our mental model of what's on the screen. What do you think?

Reply:
You right. We did the change